spotbugs/build.gradle

```diff
@@ -361,6 +352,26 @@ tasks.named('jacocoTestReport', JacocoReport).configure {
 tasks.named('spotbugsTest').configure {
   ignoreFailures = true
 }
+
+dependencies {
+  constraints {
+    logBinding("org.apache.logging.log4j:log4j-core") {
+      version {
+        strictly("[2.17.1, 3[")
+        prefer("2.17.1")
+      }
+      because("CVE-2021-44228, CVE-2021-45046, CVE-2021-45105, CVE-2021-44832: Log4j vulnerable to remote code execution and other critical security vulnerabilities")
+    }
+    logBinding("ch.qos.logback:logback-core") {
+      version {
+        strictly("[1.2.9, 2[")
+        prefer("1.2.10")
+      }
+      because("CVE-2021-42550: Logback vulnerable to remote code execution vulnerabilities")
+    }
+  }
```

Comment:
These config will keep SpotBugs away from buggy logging libraries.
https://blog.gradle.org/log4j-vulnerability

Reply:
I'm on it. I see few issues with Eclipse support also before this patch (wrong JLS settings, duplicated classpath elements, not compilable code, Java 11 dependencies introduced).